src/main/java/homework_2/PyramidPrinter.java

```diff
@@ -0,0 +1,34 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class PyramidPrinter {
```

Comment:
Good. Simple enough and handles edge cases. You could also move some parts to methods, but as-is looks good